### ISSUE TITLE
[BUG]: '"@pdanpdan/vue-keyboard-trap"' has no exported member named 'VueKeyboardTrapDirectiveFactory'

### ISSUE THREAD

**Issue by cmcleese** [pdanpdan/vue-keyboard-trap#4]
**Describe the bug**
No import possible as described in example

**To Reproduce**
```js
import { VueKeyboardTrapDirectiveFactory } from '@pdanpdan/vue-keyboard-trap';
```
**Error**: '"@pdanpdan/vue-keyboard-trap"' has no exported member named 'VueKeyboardTrapDirectiveFactory'

**Expected behavior**
Should be able to import
"@pdanpdan/vue-keyboard-trap": "^1.0.12",

**Comment by cmcleese** [comment]
There is no TS interface for `VueKeyboardTrapDirectiveFactory`

**Comment by pdanpdan** [comment]
If you know what Typescript needs in order to see the export then please tell me or make a PR

**Comment by pdanpdan** [comment]
:D I found what needs to be done. fixed in v1.0.14

**Comment by cmcleese** [comment]
Yes thats it! thanks
